src/dispatch/case/flows.py

```diff
@@ -384,3 +384,41 @@ def case_to_incident_escalate_flow(
         description=f"The members of the incident's tactical group {incident.tactical_group.email} have been given permission to access the case's storage folder",
         case_id=case.id,
     )
+
+
+def case_escalate_flow(case_id: PrimaryKey, incident_id: PrimaryKey, db_session: SessionLocal):
```

Comment:
Should we rename it to `case_to_incident_escalated_custom_flow` to denote that it's a custom version of `case_to_incident_escalate_flow`?